PWGHF/D2H/Tasks/taskDstarToD0Pi.cxx

```diff
@@ -42,12 +44,16 @@ struct HfTaskDstarToD0Pi {
   using CandDstarWSelFlagMcRec = soa::Join<aod::HfD0FromDstar, aod::HfCandDstars, aod::HfSelDstarToD0Pi, aod::HfCandDstarMcRec>;
   using CandDstarMcGen = soa::Join<aod::McParticles, aod::HfCandDstarMcGen>;
 
+  using CollisionsWCent = soa::Join<aod::Collisions, aod::CentFT0Ms>;
+  using CollisionsWCentMcLabel = soa::Join<aod::Collisions, aod::McCollisionLabels, aod::CentFT0Ms>;
```

Comment:
It is ok to use `CentFT0Ms` as default for pp collisions, but it would also be good in the future to be able to change estimator (for instance `CentFT0Cs` has to be used in Pb-Pb). For this you could templetise the process function and have multiple process functions for the different centrality estimators.
No need to do it now, but just keep it in mind for the future.

Reply:
Thanks Fabrizio.